pkg/gadgets/snapshot/socket/types/socket-collector.go

```diff
@@ -60,25 +62,26 @@ func GetColumns() *columns.Columns[Event] {
 		col.Visible = false
 	}
 
-	cols.MustAddColumn(columns.Attributes{
-		Name:     "local",
-		MinWidth: 21, // 15(ipv4) + 1(:) + 5(port)
-		MaxWidth: 51, // 45(ipv4 mapped ipv6) + 1(:) + 5(port)
-		Visible:  true,
-		Order:    1000,
-	}, func(e *Event) string {
-		return fmt.Sprintf("%s:%d", e.LocalAddress, e.LocalPort)
-	})
-
-	cols.MustAddColumn(columns.Attributes{
-		Name:     "remote",
-		MinWidth: 21, // 15(ipv4) + 1(:) + 5(port)
-		MaxWidth: 51, // 45(ipv4 mapped ipv6) + 1(:) + 5(port)
-		Visible:  true,
-		Order:    1001,
-	}, func(e *Event) string {
-		return fmt.Sprintf("%s:%d", e.RemoteAddress, e.RemotePort)
-	})
+	eventtypes.MustAddVirtualL4EndpointColumn(
+		cols,
+		columns.Attributes{
+			Name:    "src",
+			Visible: true,
+			Width:   40,
+			Order:   1000,
```

Comment:
I'm wondering if we should have a `src` and `dst` templates for this.

Reply:
We should definitely do that +1